app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
       <span class="close-cross mobile-only">&times;</span>
       <menu>
         <% if signed_in? %>
+          <%= render "shared/search_dropdown" %>
```

Comment:
should maybe comment out this and leave the search menu in for now until we design up the new search?